server/backend/quiz/admin.py

```
@@ -0,0 +1,32 @@
+
+class UserAnswerInline(admin.TabularInline):
+    model = UserAnswer
+    readonly_fields = ('question', 'selected_option', 'is_correct')
+    can_delete = False
+    extra = 0
+
+class QuizAttemptAdmin(admin.ModelAdmin):
+    list_display = ('user', 'category', 'score', 'started_at', 'completed_at')
+    list_filter = ('category', 'user')
+    search_fields = ('user__username',)
+    inlines = [UserAnswerInline]
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)